README.md

```diff
@@ -54,15 +54,19 @@ pushd kythe-verification; stack install && ./test.sh; popd
 
 # Demo
 
-To index a few packages and serve the index, run:
+To index a few packages, run:
 
-```
+```bash
 ./build-stack.sh /tmp/logs mtlparse cpu
 ```
 
-The script temporarily replaces the system GHC with
-`wrappers/stack-docker/fake-stack/ghc` script, does the indexing and serves the
-built index at `localhost:8080`.
+The script installs a post hook to `stack path --compiler-exe`, does the indexing when `ghc --make` is detected to build a package. You can run `build-stack.sh` multiple times.
```

Comment:
About using system ghc: I'm not sure the post-hook approach is more elegant (given it has to temporary modify the stack compiler script), vs '--system-ghc', which takes the original stack compiler that we prepended to the path.

But I can be convinced - the post-hooks looks nicer, except for the compiler script tampering.